commands/twitter.js

```diff
@@ -26,17 +26,21 @@ export class Twitter extends Command {
     });
 
     return new Promise((resolve, reject) => {
-      twitter.getTweet({id: tweet_id},
+      twitter.getTweet({id: tweet_id, include_entities: true},
         (error) => {
           console.log(`ERROR: Twitter Info - ${error}`);
           reject();
         },
         (success) => {
           let data = JSON.parse(success);
+          let text = data.text;
+          data.entities.urls.reverse().forEach(url => {
```

Comment:
Yeah. I don't know of a way to swap all the links at once, so i go from the end of the string to the beginning so the indices in the twitter response stay accurate.

The results from twitter are in the order they appear in the tweet, but I don't think there's actually a guarantee of that. I think it's safe to assume they'll be in order though.